src/lib/aws/aws_naming.py

```diff
@@ -79,6 +79,16 @@ def TimestreamTable(cls, stack_obj: object, meaning: str) -> str:
         prefix = "tstable"
         outp = f"{prefix}-{meaning}" # Table lives inside DB, so we identify project and stage names by DB
         return outp
+        
+    @classmethod
+    def EC2(cls, stack_obj: object, meaning: str) -> str:
+        prefix = "ec2"
+        return AWSNaming.__resource_name_with_check(stack_obj, prefix, meaning)
+    
+    @classmethod
+    def SM(cls, stack_obj: object, meaning: str) -> str:
```

Comment:
Let's call it "secret" :)
For me, it required to search through code to understand what does SM mean :)

Reply:
sure :)

Reply:
updated accordingly